lib/core/src/model.rs

```diff
@@ -76,6 +80,7 @@ impl Config {
             network: LiquidNetwork::Testnet,
             payment_timeout_sec: 15,
             zero_conf_min_fee_rate_msat: DEFAULT_ZERO_CONF_MIN_FEE_RATE_TESTNET,
+            sync_service_url: BREEZ_SYNC_SERVICE_URL.to_string(),
```

Comment:
Using the same service URL for mainnet/testnet, is this going to cause a conflict if using the same mnemonic for both? I imagine it will merge swap data from both together...